src/referencePackages/Directory.Build.props

```diff
@@ -1,6 +1,11 @@
 <Project>
 
   <Import Project="..\..\Directory.Build.props" />
+  
+  <ItemGroup>
+    <!-- Attributes for all Reference Packages -->
+    <AssemblyMetadata Include="source" Value="source-build-reference-packages" />
```

Comment:
I think we should add this to the targeting packs as well.  We shouldn't be shipping any of them.  The text-only packages  should not get this (just as you are proposing)

Reply:
Agreed, makes sense. Maybe it's fine to just duplicate this logic in both Directory.Build.props files. It won't really change anytime in the future (hopefully).